rflx/model.py

```diff
@@ -1094,7 +1493,14 @@ def qualified_type_name(name: StrID, package: ID) -> ID:
 }
 
 BOOLEAN = Enumeration(
-    BUILTINS_PACKAGE * "Boolean", {"False": Number(0), "True": Number(1)}, Number(1), False
+    BUILTINS_PACKAGE * "Boolean",
+    {
+        ID("False", Location((1, 18), Path(str(BUILTINS_PACKAGE)), (1, 21))): Number(0),
+        ID("True", Location((1, 24), Path(str(BUILTINS_PACKAGE)), (1, 28))): Number(1),
+    },
```

Comment:
Nothing, really. The `FILE:line:col` is necessary to have the output accepted by IDEs. I changed it to `0:0` in 408937b9bde04a1441df2833ec1a396edc409543 to avoid the impression that this has any particular meaning.